mobile/lib/features/trade/channel_configuration.dart

```diff
@@ -144,156 +132,142 @@ class _ChannelConfiguration extends State<ChannelConfiguration> {
         (maxCounterpartyCollateral.sats * counterpartyLeverage).toInt();
 
     return Form(
-        key: _formKey,
+      key: _formKey,
+      child: SingleChildScrollView(
```

Comment:
I guess the better solution would be to reduce the content on this modal. This could be a blocker for users already because it's too complicated, now the might even have to scroll 🙈 

Can't we reduce the text a bit?

Reply:
I reduced the text a bit.